### PR TITLE
Visualization function

### PR DESCRIPTION
**Pull request by tonyshumlh**

### CODE REVIEW

File: src/gic_calculator/gic_plotting.py
Comment (AlysenTownsley) [inline src/gic_calculator/gic_plotting.py:1]: We may want to include more comments to explain the steps of this function (similar to what you have done for the test file).

**Comment by tonyshumlh** [comment]
Have to double check y-axis format in the Chart

File: src/gic_calculator/gic_plotting.py
Comment (AlysenTownsley) [reply src/gic_calculator/gic_plotting.py:1]: Looks good, thanks for adding the comments.

**Review by AlysenTownsley — APPROVED** [review]

**Comment by AlysenTownsley** [comment]
Reviewed and approved the function and additional comments

**Comment by sungg888** [comment]
The function is good to merge.

**Pull request merged**